src/main/java/frc/robot/subsystems/ramp/RampIOSim.java

```diff
@@ -0,0 +1,32 @@
+package frc.robot.subsystems.ramp;
+
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.system.plant.DCMotor;
+import edu.wpi.first.wpilibj.simulation.SingleJointedArmSim;
+
+// TODO make pid constants
+public class RampIOSim implements RampIO {
+
+  private PIDController controller = new PIDController(7.5, 0.001, 0.0);
+
+  private SingleJointedArmSim sim =
+      new SingleJointedArmSim(
+          DCMotor.getKrakenX60(1), 25, 10, 24.938, 0, 2 * Math.PI, true, Math.PI);
```

Comment:
Can we make these magic numbers into named constants? It would be nice if they were JSON + used units when applicable as well